lib/rules/jsx-indent-props.js

```diff
@@ -173,16 +173,22 @@ module.exports = {
      */
     function checkNodesIndent(nodes, indent) {
       nodes.forEach((node) => {
+        let nestedIndent = indent;
```

Comment:
```suggestion
      let nestedIndent = indent;
      nodes.forEach((node) => {
```
otherwise it isn't persisted across the forEach